<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1047</width>
    <height>762</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Gauche">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="avancer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>440</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Avancer</string>
    </property>
   </widget>
   <widget class="QPushButton" name="droite">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>500</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Doite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="reculer">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>560</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Reculer</string>
    </property>
   </widget>
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>100</y>
      <width>411</width>
      <height>281</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="camera"/>
   </widget>
   <widget class="QPushButton" name="hgauche">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Haut/Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="hdroite">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>470</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Haut/Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bgauche">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bas/Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bdroite">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>530</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bas/Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="connexion">
    <property name="geometry">
     <rect>
      <x>520</x>
      <y>440</y>
      <width>161</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connexion</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="barre_batterie">
    <property name="geometry">
     <rect>
      <x>790</x>
      <y>70</y>
      <width>241</width>
      <height>71</height>
     </rect>
    </property>
    <property name="value">
     <number>23</number>
    </property>
   </widget>
   <widget class="QLabel" name="titre_batterie">
    <property name="geometry">
     <rect>
      <x>890</x>
      <y>40</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Batterie</string>
    </property>
   </widget>
   <widget class="QSlider" name="slide_vitesse">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>590</y>
      <width>281</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>120</number>
    </property>
    <property name="maximum">
     <number>240</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="titre_vitesse">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>560</y>
      <width>111</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse</string>
    </property>
   </widget>
   <widget class="QPushButton" name="haut_camera">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>110</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Haut</string>
    </property>
   </widget>
   <widget class="QPushButton" name="gauche_camera">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QPushButton" name="droite_camera">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>140</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bas_camera">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>170</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Bas</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_ir">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Infrarouge</string>
    </property>
   </widget>
   <widget class="QLabel" name="ir_AvG">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Avant Gauche</string>
    </property>
   </widget>
   <widget class="QLabel" name="ir_AvD">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>200</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Avant Droite</string>
    </property>
   </widget>
   <widget class="QLabel" name="ir_ArG">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arriere Gauche</string>
    </property>
   </widget>
   <widget class="QLabel" name="ir_ArD">
    <property name="geometry">
     <rect>
      <x>970</x>
      <y>240</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Arriere Droite</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_odo">
    <property name="geometry">
     <rect>
      <x>910</x>
      <y>280</y>
      <width>61</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Odométrie</string>
    </property>
   </widget>
   <widget class="QLabel" name="odo_gauche">
    <property name="geometry">
     <rect>
      <x>840</x>
      <y>310</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Gauche</string>
    </property>
   </widget>
   <widget class="QLabel" name="odo_droite">
    <property name="geometry">
     <rect>
      <x>980</x>
      <y>310</y>
      <width>47</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>Droite</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_vit">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>620</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Vitesse en cm/s</string>
    </property>
   </widget>
   <widget class="QLabel" name="collision">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>400</y>
      <width>201</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Recherche de collision</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_camera">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>70</y>
      <width>121</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouvement de la caméra</string>
    </property>
   </widget>
   <widget class="QLabel" name="titre_mouvement">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>400</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Mouvement du robot</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1047</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
